.github/workflows/daily-integration-check.yml

```diff
@@ -34,3 +34,24 @@ jobs:
         with:
           filename: .github/integration-check-failure-issue.md
           update_existing: true
+
+  load-testing-check:
```

Comment:
should we have this create a github issue as well?

Reply:
>  should we have this create a github issue as well?

Not yet because we don't even know what to do if it fails. For example, at the moment we have no idea why the performance is fluctuating. At the moment i view this check as a "monitor" more than a "check".

Reply:
ah ok makes sense

Reply:
I see us integrating this with https://asv.readthedocs.io/en/stable/ in the far future.

Reply:
another weird question: should we set this to run at a slightly different time than the other integration tests? otherwise they'll impact the server load

Reply:
People will still be making requests to Superstaq anyway when we are load testing no? We could set a window that says "no one can make a request to Superstaq except the load tester for *this period of time" but i think that would be overkill.